README.md

```diff
@@ -1,21 +1,21 @@
 # Orka Images
 
-Repository with Orka 3.0 OCI-compatible images. 
+Repository with Orka 3.2 OCI-compatible images. 
 
-These images can be used with Orka Desktop (via the GUI), or Orka Cluster (via the orka3 CLI)
+These images can be used with Orka Desktop (via the GUI), or with Orka via the [orka3 CLI](https://orkadocs.macstadium.com/docs/cli-reference)
 
 ## Using with Orka Desktop
-To get started with Orka Desktop, click the 'Create New VM', choose 'Pull from Image', and use the following OCI image name:
+To get started with [Orka Desktop](https://github.com/macstadium/orka-desktop), click the 'Create New VM', choose 'Pull from Image', and use the following OCI image name:
 
 ```sh
-ghcr.io/macstadium/orka-images/sonoma:latest
+ghcr.io/macstadium/orka-images/sequoia:latest
```

Comment:
The Sequoia image won't work with Desktop right now. We need to release Desktop with the support for the new OCI image format

We can mention this explicitly for now

Reply:
@spikeburton removed the Sequoia mentions and made a note mentioning that the support for the new image format is coming in the next release explicitly!

Reply:
Should we explicitly mention that the Sequoia image is in the new format and that this is why for Desktop we're showing the Sonoma image?

Just don't want folks to be confused. They may not know what "new OCI image format" means